keras_core/layers/preprocessing/random_flip_test.py

```diff
@@ -67,6 +67,28 @@ def test_random_flip_vertical(self):
             run_training_check=False,
         )
 
+    def test_support_jit(self):
```

Comment:
As far as I can tell this is already being tested in `run_layer_test`, we don't need to test it here. One thing you can do to check it's actually being tested is add a `raise` in the `@jax.jit` functions in `backend/jax/trainer.py` and run the tests -- they should fail since the code path will be taken.

Reply:
Hi @fchollet 
Actually, we must set `run_training_check=True` to test the jit functionality. I have updated the tests.